libstuff/SX509.h

```diff
@@ -5,11 +5,17 @@
 // Wraps a X509 Cert
 struct SX509 {
     // Attributes
-    mbedtls_x509_crt srvcert;
+    mbedtls_x509_crt cert;
     mbedtls_pk_context pk;
 };
 
 // X509 Certificates
 extern SX509* SX509Open();
+
 extern SX509* SX509Open(const string& pem, const string& srvCrt, const string& caCrt);
+
+extern SX509* SX509Open(const string& casCrt);
```

Comment:
I don't think there's a good reason for these to be `extern`. Since we're touching them now, maybe we can fix that.